dojoWordle/components/LetterBox.tsx/LetterBox.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Text } from "react-native";
+import { Box } from "./LetterBox.style";
+
+type letterStatus = "success" | "failure" | "missed";
```

Comment:
```suggestion
type LetterStatus = "success" | "failure" | "missed";
```

Use uppercase for types